src/client.ts

```diff
@@ -601,9 +601,9 @@ interface IJoinRequestBody {
     third_party_signed?: IThirdPartySigned;
 }
 
-interface ITagMetadata {
+export interface ITagMetadata {
```

Comment:
why is this changing, out of interest?

Reply:
That should not have ended up here, removed 🔥